src/provider/application.js

```diff
@@ -260,8 +260,15 @@ class Application extends EventEmitter {
   }
 
   unload() {
+    // Need to check for href attributes to avoid flickering.
+    const attributeCheck = document.activeElement.getAttribute('href');
+    let attributeCheck_href = false;
+    if(attributeCheck.includes('tel') || attributeCheck.includes('mailto') || attributeCheck.includes('fax') || 
+    attributeCheck.includes('sms') || attributeCheck.includes('callto')) {
```

Comment:
I think this would be clearer to keep the invalid schemes in an array, then check the href's scheme for membership.

```javascript
const invalidSchemes = ['sms', 'tel'];
const scheme = tag.attributes['href'].split(':')[0];
const hasInvalidScheme = invalidSchemes.includes(scheme)
```

Reply:
with the new code change, this won't be necessary